pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java

```diff
@@ -51,7 +51,7 @@
 /**
  * Generate metrics aggregated at the namespace level and optionally at a topic level and formats them out
  * in a text format suitable to be consumed by Prometheus.
- * Format specification can be found at {@link https://prometheus.io/docs/instrumenting/exposition_formats/}
+ * Format specification can be found at {@link: https://prometheus.io/docs/instrumenting/exposition_formats/}
```

Comment:
The link is broken in the java doc.
<img width="713" alt="image" src="https://user-images.githubusercontent.com/16974619/160343030-c83690dc-635f-45ae-96e6-20734c31c1a6.png">